File: BE/src/main/java/com/backend/eyeson/dto/ResponseCompDto.java
package com.backend.eyeson.dto;

import com.backend.eyeson.entity.CompStateEnum;
import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ResponseCompDto {
    private long compSeq;
    private String compAddress;
    private String compContent;
    private String compImage;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm", timezone = "Asia/Seoul")
    private LocalDateTime compRegtime;
    private String compResultContent;
    private String compReturn;
    private CompStateEnum compState;
    private String compTitle;

}

File: AOS/EyesOn/data/src/main/java/com/d201/data/model/response/ResponseCompDto.kt
package com.d201.data.model.response

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class ResponseCompDto(
    @SerializedName("compSeq") val seq: Long,
    @SerializedName("compAddress") val address: String?,
    @SerializedName("compContent") val content: String?,
    @SerializedName("compImage") val image: String?,
    @SerializedName("compRegtime") val regTime: String?,
    @SerializedName("compResultContent") val resultContent: String?,
    @SerializedName("compReturn") val returnContent: String?,
    @SerializedName("compState") val state: String?,
    @SerializedName("compTitle") val title: String?
)